EC2/datasource.tf

```diff
@@ -0,0 +1,30 @@
+
+# Get Latest AWS AMI ID for Amazon2 Linux
+data "aws_ami" "Latest_Image" {
+   most_recent      = true
+  owners           = ["099720109477"]
```

Comment:
What's the use of owners? what it is representing?

Reply:
Owner represent that the latest ami-id we have fetched from AWS belongs to that owner.There is lots of ami-id attached to this owner , but we fetched the latest ubuntu ami-id  of that owner